x-pack/plugins/alerting/server/saved_objects/migrations/index.test.ts

```diff
@@ -2509,37 +2509,91 @@ describe('successful migrations', () => {
-      const migration870 = getMigrations(encryptedSavedObjectsSetup, {}, isPreconfigured)['8.7.0'];
-      const rule = getMockData(
-        {
-          params: { esQuery: '{ "query": "test-query" }', searchType: 'esQuery' },
-          alertTypeId: '.es-query',
-        },
-        true
-      );
-      const migratedAlert870 = migration870(rule, migrationContext);
+    describe('es_query rule', () => {
+      test('migrates es_query rule params and adds group by fields', () => {
+        const migration870 = getMigrations(encryptedSavedObjectsSetup, {}, isPreconfigured)[
+          '8.7.0'
+        ];
+        const rule = getMockData(
+          {
+            params: { esQuery: '{ "query": "test-query" }', searchType: 'esQuery' },
+            alertTypeId: '.es-query',
+          },
+          true
+        );
+        const migratedAlert870 = migration870(rule, migrationContext);
 
-      expect(migratedAlert870.attributes.params).toEqual({
-        esQuery: '{ "query": "test-query" }',
-        searchType: 'esQuery',
-        aggType: 'count',
-        groupBy: 'all',
+        expect(migratedAlert870.attributes.params).toEqual({
+          esQuery: '{ "query": "test-query" }',
+          searchType: 'esQuery',
+          aggType: 'count',
+          groupBy: 'all',
+        });
+      });
+
+      test('does not migrate rule params if rule is not es query', () => {
+        const migration870 = getMigrations(encryptedSavedObjectsSetup, {}, isPreconfigured)[
+          '8.7.0'
+        ];
+        const rule = getMockData(
+          {
+            params: { foo: true },
+            alertTypeId: '.not-es-query',
+          },
+          true
+        );
+        const migratedAlert870 = migration870(rule, migrationContext);
+
+        expect(migratedAlert870.attributes.params).toEqual({ foo: true });
       });
     });
 
-    test('does not migrate rule params if rule is not es query', () => {
-      const migration870 = getMigrations(encryptedSavedObjectsSetup, {}, isPreconfigured)['8.7.0'];
-      const rule = getMockData(
-        {
-          params: { foo: true },
-          alertTypeId: '.not-es-query',
+    describe('log threshold rule', () => {
+      const logThresholdAlertTypeId = 'logs.alert.document.count';
+
+      const params = {
+        timeSize: 5,
+        timeUnit: 'm',
+        count: {
+          value: 75,
+          comparator: 'more than',
         },
-        true
-      );
-      const migratedAlert870 = migration870(rule, migrationContext);
+        criteria: [
+          {
+            field: 'log.level',
+            comparator: 'equals',
+            value: 'error',
+          },
+        ],
+      };
 
-      expect(migratedAlert870.attributes.params).toEqual({ foo: true });
+      const logView = {
+        logViewId: 'log-view-reference-0',
+        type: 'log-view-reference',
+      };
+
+      test('should migrate and add the logView param', () => {
```

Comment:
Thanks for also adding these tests! Do we also want to test that the reference is added correctly?